package.json

```diff
@@ -76,16 +76,18 @@
     "karma-coverage-istanbul-reporter": "^3.0.3",
     "karma-jasmine": "~4.0.0",
     "karma-jasmine-html-reporter": "^1.5.0",
+    "karma-junit-reporter": "^2.0.1",
     "protractor": "^7.0.0",
     "replace-in-file": "^6.2.0",
     "tar": "^6.1.1",
     "ts-node": "~10.1.0",
-    "typescript": "4.3.5",
+    "typescript": ">= 4.2.4 <= 4.3.5",
     "url-parse": ">=1.5.3"
   },
   "snyk": true,
   "engines": {
     "npm": "please-use-yarn",
+    "node": ">=14.15.5 <15",
```

Comment:
I believe this should be the single source of truth for the node version, and  it should feed other locations, such as gitlab-ci.
What do you think @MightyAx?

Reply:
I agree that a single source of truth would be better but I can't think of a way we could do that.